packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -18,8 +22,8 @@ import { boundWorldPointToFOV, getModifierKeyArray } from './utils';
 interface Props extends CommonInteractionProps {
   id?: string;
   transformSelection?: (selection: Selection) => Selection;
-  onSelectionStart?: (evt: CanvasEvent<PointerEvent>) => void;
-  onSelectionChange?: (selection: Selection) => void;
+  onSelectionStart?: (selection: Selection) => void;
```

Comment:
Do you remember why we had the `evt` as parameter here instead of `selection` ?

Also, it is fortunate that we derive several interfaces from these `Props` (I was wondering why you didn't need to change anything in `AxialSelectionTool` :smile: )

Reply:
Hehe, yeah it's good. I had added `evt` in a previous PR in case we needed it in the future or in case a consumer needed it, but we don't actually do need it yet. Now, by the time this is called, a `selection` object is already computed, so it makes sense to expose it I think.